cc-pk-service/.env.example

```diff
@@ -1,5 +1,17 @@
+<<<<<<< HEAD
```

Comment:
Remove all the merge conflicts from this file

Reply:
This was odd, but fixed it. Would be easier to work with only own code. Hopefully I have not messed up anything in other parts :D